cmake/std/atdm/ride/all_supported_builds.sh

```diff
@@ -1,13 +1,6 @@
 export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-white-ride-
 
 export ATDM_CONFIG_ALL_SUPPORTED_BUILDS=(
-  cuda-9.2-gnu-7.2.0-debug
-  cuda-9.2-gnu-7.2.0-release
-  cuda-9.2-gnu-7.2.0-release-debug
-  gnu-7.2.0-openmp-debug
-  gnu-7.2.0-openmp-release
-  gnu-7.2.0-openmp-release-debug
-  cuda-10.1-gnu-7.2.0-debug
-  cuda-10.1-gnu-7.2.0-release
-  cuda-10.1-gnu-7.2.0-release-debug
```

Comment:
No, you have to put these back while we are still running builds on 'ride'.  As long as 'ride' builds are posting to CDash, those builds have to be listed here.